enterprise/backend/test/metabase_enterprise/workspaces/api_test.clj

```diff
@@ -294,9 +322,10 @@
             ;; create the target table
             (transforms.i/execute! x1 {:run-method :manual})
             (let [;; create the workspace
-                  workspace (mt/user-http-request :crowberto :post 200 "ee/workspace"
-                                                  {:name        "Test Workspace"
-                                                   :database_id (mt/id)})
+                  workspace (u/prog1 (mt/user-http-request :crowberto :post 200 "ee/workspace"
```

Comment:
I see you do this `u/prog1` + `ws-ready` combo together a lot. 

Since `ws-ready` returns the latest workspace, why not just `(ws-ready (mt/user-http-request ...))`?

Maybe in some cases you care about the response while it's still processing, but it doesn't look like it in many others.